MekHQ/src/mekhq/utilities/RandomUtils.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) 2024 - The MegaMek Team. All Rights Reserved.
+ *
+ *  This file is part of MekHQ.
+ *
+ *  MekHQ is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  MekHQ is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with MekHQ. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package mekhq.utilities;
+
+import megamek.common.Compute;
+
+import java.util.List;
+import java.util.Optional;
+
+;
+public class RandomUtils {
```

Comment:
What benefit does this give me over `ObjectUtility.getRandomItem()`? And, is there value in moving this into that class, instead of having two separate classes that, on the surface, both seem to be doing the same thing.

Reply:
I wasant aware of that function. But the benefit is the Optional, which does allow for some safer object manipulation as the null test is a bit harder to get around.